app/db/models/fund.py

```diff
@@ -43,10 +44,12 @@ class Fund(BaseModel):
         default=uuid.uuid4,
         nullable=False,
     )
-    name_json = Column("name_json", JSON(none_as_null=True), nullable=False, unique=False)
-    title_json = Column("title_json", JSON(none_as_null=True), nullable=False, unique=False)
+    name_json = Column("name_json", MutableDict.as_mutable(JSON(none_as_null=True)), nullable=False, unique=False)
```

Comment:
Did we use `MutableDict.as_mutable` to track changes of the data ? if we are using this I think there are few drawbacks mainly performance impact if we are frequently going to update these columns. 

Same for all the columns..

Reply:
Discussed with Nuwan, agreed here it makes sense for these to be mutable.